types/session_cost.yaml

```diff
@@ -0,0 +1,79 @@
+description: Types to provide the session price
+types:
+  SessionCostChunk:
+    description: >-
+      A chunk of the session cost. The total cost of the session is the sum of
+      all chunks.
+    type: object
+    additionalProperties: false
+    properties:
+      timestamp_from:
+        description: >-
+          Absolute timestamp for the start of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      timestamp_to:
+        description: >-
+          Absolute timestamp for the end of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      cost:
+        description: >-
+          Cost of the energy consumed during this chunk in the currency
```

Comment:
Cost of the energy consumed (or other things like time, base price, ...) during this chunk in the currency